package.json

```diff
@@ -20,11 +20,14 @@
     "parse-link-header": "^1.0.1"
   },
   "devDependencies": {
-    "chai": "^4.2.0",
-    "eslint": "^7.10.0",
-    "eslint-config-prettier": "^6.12.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "mocha": "^8.1.3",
-    "prettier": "^2.1.2"
+    "eslint": "^6.8.0",
```

Comment:
Our dependencies have been updated. Could you update yours to match?

Reply:
Should be up to date now